code/Equipment/Weapons/TeleportToolComponent.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Grubs.UI.Components;
+using System.Diagnostics;
+
+namespace Grubs.Equipment.Weapons;
+
+[Title( "Grub - Teleport Tool" ), Category( "Equipment" )]
+public class TeleportToolComponent : WeaponComponent
+{
+	/*
+	 * Cursor Properties
+	 */
+
+	[Property] public float CursorRange { get; set; } = 100f; // How far from the grub can the tool target
```

Comment:
Unnecessary space

Reply:
Fixed in https://github.com/apetavern/grubs/pull/355/commits/23dbbb2ef5dd9dddee2079e4c739e9fe0dcb0a3c